docs/05.PORT-API.md

```diff
@@ -0,0 +1,287 @@
+# Reference
+
+## Termination
+
+It is questionable whether a library should be able to terminate an application. However, as of now, we only have the concept of completion code around `jerry_parse` and `jerry_run`. Most of the other API functions have no way of signaling an error. So, we keep the termination approach with this port function.
```

Comment:
I mean, completion code is an old concept, which has been removed, AFAIK. Those functions return jerry_value_t nowadays. So that seems definitely like an outdated sentence.